<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FightWindow</class>
 <widget class="QWidget" name="FightWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#FightWindow{

}</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>611</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="attacker_image">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>158</width>
        <height>226</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="enemy_image">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="skill">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#skill{
color:rgb(26, 60, 77);
font-size:30px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>611</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="attacker_Hurt">
      <property name="styleSheet">
       <string notr="true">#attacker_Hurt{
color:red;
font-size:18px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="enemy_Hurt">
      <property name="styleSheet">
       <string notr="true">#enemy_Hurt{
color:red;
font-size:18px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="fight_start">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#fight_start{
color:rgb(32, 59, 10);
font-size:35px;
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.506, fy:0.966, stop:0 rgba(0, 0, 0, 0), stop:0.275281 rgba(159, 235, 148, 64), stop:0.340225 rgba(0, 0, 0, 0), stop:0.565 rgba(82, 121, 76, 33), stop:0.721925 rgba(255, 238, 150, 129), stop:0.77 rgba(155, 255, 174, 204), stop:0.89 rgba(191, 128, 255, 64), stop:1 rgba(0, 0, 0, 0))
}</string>
   </property>
   <property name="text">
    <string>Fight! Start!</string>
   </property>
  </widget>
  <widget class="QLabel" name="fight_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#fight_back{
background-image: url(:/resource/back7.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>fight_back</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>skill</zorder>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>fight_start</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
